external/slowparse/slowparse.js

```diff
@@ -1629,15 +1629,15 @@
     // passed the beginning `<tagname` of an HTML element.
     _parseCDATA: function(tagname) {
       var token,
-           matchString = '</'+tagname+'>',
+           matchString = `</${tagname}>`,
```

Comment:
Template literals are nice, but let's keep this PR to just the code that's needed to get this to work (versus ES upgrades).
I think it's possible we might want to merge our slowparse changes with the upstream slowparse changes, so it's best to restrict our changes to what's needed.

Reply:
Understood.